components/tag-list/tag-list.js

```diff
@@ -214,7 +210,7 @@ class TagList extends LocalizeCoreElement(ArrowKeysMixin(LitElement)) {
 
 			for (let i = overflowingIndex; i < this._itemLayouts.length; i++) {
 				const itemLayout = this._itemLayouts[i];
-				const itemWidth = Math.min(itemLayout.width, this._availableWidth);
+				const itemWidth = Math.min(itemLayout.width + MARGIN_TOP_RIGHT, this._availableWidth);
```

Comment:
I am going to rename this const to `GAP` - just including it with my interactive changes to avoid conflicts.